apps/prairielearn/src/lib/chunks.ts

```diff
@@ -477,10 +497,12 @@ export async function diffChunks({
         existingCourseChunks[courseChunk.type] = true;
         break;
       case 'question':
+        assert(courseChunk.question_name != null, 'question_name is required');
```

Comment:
These asserts in `chunks.ts` are probably the most dangerous change, as most changes are to the test suite.